torchreid/utils/tools.py

```diff
@@ -28,7 +28,7 @@
 ]
 
 def get_git_revision():
-    path = os.getcwd()
+    path = os.path.abspath(os.path.dirname(__file__)) + '/../..' # two folders up to .git folder
     sha_message = ['git', '--git-dir', f'{path}/.git', 'rev-parse', 'HEAD']
```

Comment:
git returns correct commit hash from any subfolder in a repo, so there is no need to add this key to the command as well as to hardcode a relative location in the line above.

Reply:
hmm, 
actually, it doesn't work without `--git-dir` folder and exact path to .git folder

Reply:
Okay, I see. Miss the -cwd parameter

Reply:
Now, it works, thanks!